npc/re/guild/mission_main.txt

```diff
@@ -0,0 +1,434 @@
+				break;
+		}
+		return;
+	}
+
+	function mob_brief {
+		switch(getarg(0)){
+			case 7525:
+				mes "You should hunt 50 or more Caterillars and Cremy Fear monsters..";
+				return 0;
+			case 7526:
+				mes "Hunt down 30 or more of all 3 types of Abyss Kobolds.";
+				return 0;
+			case 7527:
+				mes "^4d4dffYou should kill furious 'Pyuriel' and warrior 'Lora'.^000000";
+				return 1;
+			case 7528:
+				mes "^4d4dff Hunt 50 or more of Giant Hornets and Ancient Worms, respectively.^000000";
+				return 0;
+			case 7529:
+				mes "^4d4dff Hunt 40 or more of Killer Mantises and 50 or more of Angra Mantises.^000000";
+				return 0;
+			case 7530:
+				mes "They are ^4d4dff 'Elvira' and 'Gioia,' which is a strange mechanical creature of heroes' tears.^000000";
+				return 1;
+			case 7531:
+				mes "^4d4dffHunt 50 or more Zombie Masters and Wraith Deads, respectively.^000000";
+				return 0;
+			case 7532:
+				mes "^4d4dffHunt 50 GLD_DARK_SHADOW, 20 GLD_DARK_FRAME, and 20 GLD_DARK_PRIEST monsters.^000000";
+				return 0;
+			case 7533:
+				mes "They are ^4d4dff'Kades' and 'Rudo' who wonder around the dead's hill.^000000";
+				return 1;
+			case 7534:
+				mes "^4d4dffHunt 50 or more Gullin Bursti and Leib Olmai, respectively.^000000.";
+				return 0;
+			case 7535:
+				mes "^4d4dffHunt 45 or more Skeleton General, 20 or more Am Mut and 25 or more Gajomart, respectively.^000000.";
+				return 0;
+			case 7536:
+				mes "^4d4dffThey are Old Empire's General'Daehyon' and Guard 'Soheon'^000000.";
+				return 1;
+			case 7537:
+				mes "^4d4dffHunt 50 or more Hell Apocalips and Zakudam, respectively.^000000.";
+				return 0;
+			case 7538:
+				mes "^4d4dffHunt 35 or more Heavy Metaling and Cobalt Mineral, respectively.^000000.";
+				return 0;
+			case 7539:
+				mes "^4d4dffHunt more than 50 Banshee Masters and 30 Beholder Masters^000000.";
+				return 0;
+			case 7540:
+				mes "^4d4dffHunt 50 Aunoes, and 20 Fanats^000000.";
+				return 0;
+			case 7541:
+				mes "^4d4dffHunt 30 or more Knight, Blacksmith, Assassin, respectively^000000.";
+				return 0;
+			case 7542:
+				mes "^4d4dffHunt 30 or more Wizard, Priest, Hunter respectively^000000.";
+				return 0;
+			case 7561:
+				mes "^4d4dffHunt 30 or more, Crusader, Rogue, Alchemist, respectively^000000.";
+				return 0;
+			case 7562:
+				mes "^4d4dffHunt 30 or more Sage, Monk, Bard, respectively^000000.";
+				return 0;
+		}
+		return 0;
+	}
+	
+	function calcexp {
+		.@maxlevel = getarg(0);
+		.@isboss = getarg(1, 0);
+		
+		if(!.@isboss && BaseLevel > .@maxlevel){
+			return 1000 * (100 + ((.@maxlevel - 100) * 3));
+		} else {
+			return 1000 * (100 + ((BaseLevel - 100) * 3)) * (2 * .@isboss);
+		}
+	}
+	
+	function calcjobexp {
+		.@hasFixedBonus = getarg(0,0);
+		.@TErestricted = getarg(1,0);
+		
+		if(.@TErestricted){
+			if(jobcanentermap("te_prtcas01")){
+				if (JobLevel > 50)
+					return 1000 * (50 + ((JobLevel - 50) * 3));
+				else
+					return 5000;
+			}
```

Comment:
Maybe a custom debugmes should be added here too(like at the beginning of the script)? Additionally to the comments of cydh and atemo for sure.